CONTRIBUTING.md

```diff
@@ -341,6 +341,7 @@ branch (*warning*: you could lose commits).
 More Information
 ----------------
 
+- [ITK Software Guide, Book 1, Part III: Development Guidelines](https://ItkSoftwareGuide.pdf)
```

Comment:
The link is broken

Reply:
Should be may be https://itk.org/ItkSoftwareGuide.pdf ?

Reply:
Fixed